cypress/support/fragments/requests/requestDetail.js

```diff
@@ -117,6 +117,14 @@ export default {
     InteractorsTools.checkKeyValue(requestInfoSection, 'Patron comments', data.comments);
   },
 
+  checkItemBarcode: (barcode) => {
+    requesterInfoSection.find(KeyValue('Item barcode', { value: barcode })).exists();
```

Comment:
I'd recommend to use the following construction:
```JS
requesterInfoSection.find(KeyValue('Item barcode')).has({ value: barcode })
```
in case of error the message will be more informative